src/main/scala/org/opennetworkinsight/LDA.scala

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
I think a better name for this Class/File would be "SuspicousConnects" or something like that

also... why is the choice of protocol (or "analysis" more generally) handled separately from the parser?